components/form/form-errory-summary.js

```diff
@@ -74,7 +74,7 @@ class FormErrorSummary extends LocalizeCoreElement(RtlMixin(LitElement)) {
 					<d2l-button-icon
 						aria-expanded=${this._expanded ? 'true' : 'false'}
 						icon=${this._expanded ? 'tier1:arrow-collapse-small' : 'tier1:arrow-expand-small' }
-						text="Error Summary"
+						text="${this.localize('components.form-error-summary.text')}"
```

Comment:
We likely want this text to be different depending on `this._expanded` - something like "Expand error summary" / "Collapse Error Summary". I'd recommend just quickly confirming this with Jeff. Sorry I totally should have caught this on Friday!

Reply:
No worries at all. I thought about that too, but at least in NVDA it's reading expanded/collapsed already, I saw what we have set in demos/tests for other collapsible components and usually we just include an explanatory label/title with them. I'll confirm with Jeff.

Reply:
Should have mentioned this sooner but Jeff approved the new text value for now, with possible changes to the accessibility of alerts in the future.